docs/standard/base-types/best-practices-display-data.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Best practices for displaying and persisting formatted data in .NET
```

Comment:
This file's content is not new - it was just split off from the uber best practices doc.